.github/workflows/build.yaml

```diff
@@ -262,8 +262,18 @@ jobs:
       - name: Build standalone release
         run: source scl_source enable devtoolset-9 && yarn release:standalone
 
+      - name: Fetch dependencies from cache
+        id: cache-node-modules
+        uses: actions/cache@v3
+        with:
+          path: "**/node_modules"
+          key: yarn-build-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            yarn-build-
```

Comment:
Alternatively we could remove just the `npm install` line and put that in the yaml.  Or make `yarn release:standalone` take an argument for whether to install.

Reply:
Good thinking! I'm going to add this to my todo list since I didn't get to it here. I'm thinking we do the approach you suggested and move the platform steps out of `build.yaml` so maybe I can tackle this with that!